src/test/common/fdc3.app-channels.ts

```diff
@@ -113,10 +113,10 @@ export function createAppChannelTests(cc: ChannelControl<any,any>, documentation
         const testChannel = await cc.createTestChannel()
         const resolveExecutionCompleteListener = cc.initCompleteListener(acTestId6)
 
-        cc.setupAndValidateListener1(testChannel, "unexpected-context", errorMessage, () => { /*noop*/ })
-        cc.unsubscribeListeners()
+        await cc.setupAndValidateListener1(testChannel, "unexpected-context", errorMessage, () => { /*noop*/ })
```

Comment:
This will work/produce the correct result (unless someone transmit context with type "unexpected-context") but the failure message will not be ideal. The util function is not buying us much in this case - it might be better to just go with a naked addContextListener and assert a failure if any context is received.